README.md

```diff
@@ -1,2 +1,21 @@
-# stockanalyzer
+# Stockanalyzer
 An application which manages and aggregates stock data
+
+## Description
+
+Stockanalyzer uses external API to retrieve stock data and persists it to an in-memory database.
```

Comment:
- Please describe, are you going to get data from 3rd-party service or it will be some random generated data?
- Describe how are you going to use technologies from list in next paragraph